packages/phoenix-event-display/src/tests/managers/ui-manager/phoenix-menu/phoenix-menu-ui.test.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * @jest-environment jsdom
+ */
+import { Object3D } from 'three';
+import { InfoLogger } from '../../../../helpers/info-logger';
+import { ThreeManager } from '../../../../managers/three-manager';
+import { PhoenixMenuNode } from '../../../../managers/ui-manager/phoenix-menu/phoenix-menu-node';
+import { PhoenixMenuUI } from '../../../../managers/ui-manager/phoenix-menu/phoenix-menu-ui';
+
+jest.mock('../../../../managers/three-manager');
+
+describe('PhoenixMenuUI', () => {
+  let phoenixMenuUI: PhoenixMenuUI;
+  let phoenixMenuUIPrivate: any;
+  let phoenixMenuRoot: PhoenixMenuNode;
+  let three: ThreeManager;
+
+  beforeEach(() => {
+    three = new ThreeManager(new InfoLogger());
+    phoenixMenuRoot = new PhoenixMenuNode('test');
+    phoenixMenuUI = new PhoenixMenuUI(phoenixMenuRoot, three);
+    phoenixMenuUIPrivate = phoenixMenuUI as any;
+  });
+
+  afterEach(() => {
+    phoenixMenuUI.clear();
+  });
+
+  it('should create an instance', () => {
+    expect(phoenixMenuUI).toBeTruthy();
+  });
+
+  it('should add geometry (detector geometry) folder to the menu', () => {
+    phoenixMenuUI.addGeometryFolder();
+
+    expect(phoenixMenuUIPrivate.geomFolder.name).toBe('Detector');
+    expect(phoenixMenuUIPrivate.geomFolder).toBeInstanceOf(PhoenixMenuNode);
+    expect(phoenixMenuUIPrivate.geomFolder.toggleState).toBeTruthy();
```

Comment:
Just assert what we expect as we mention it in the test case statement. We don't need to check each property of the object. This is true for all the other tests you wrote as well.